core/src/main/scala/cromwell/core/io/IoCommand.scala

```diff
@@ -24,18 +31,41 @@ object IoCommand {
 }
 
 trait IoCommand[+T] {
+  private val uuid = UUID.randomUUID().toString
+  private val creation = OffsetDateTime.now
+
+  def commandDescription: String
+
+  def logIOMsgOverLimit(message: String): Unit = {
+    val millis = java.time.Duration.between(creation, OffsetDateTime.now).toMillis
+    val seconds = millis / 1000D
+    if (seconds > 5 * 60) {
+      IoCommand.logger.info(f"(IO-$uuid) '$message' is over 5 minutes. It has been running for " +
```

Comment:
Since AFAICT this only logs on completion, the `"has been running for..."` could probably be more accurately `"was running for..."`?